### ISSUE TITLE
Update nuget package

### ISSUE THREAD

**Issue by tony56a** [Archomeda/csl-ambient-sounds-tuner#40]
Hi,

I'm looking to set up appvoyer for my project, when I came across your nuget packages for the C:S internal libraries. However, I did notice though that the libraries don't seem to been updated for the latest released version( 1.4.0-f3 ), would it be possible to update them? ( Provided that I'm not gonna get in trouble for  possible [violations of the EULA](http://store.steampowered.com/eula/202090_eula_0) :laughing: )

P.S: Thanks so much for providing the service! This seems like a really great idea, maybe you should ask Colossal Order/Paradox to maintain it, if possible?

**Comment by macronomicus** [comment]
yeah ive noticed a few errors popping up, ill miss the good sounds for now, I dont believe you'll violate anything right? This is in github as opensource so you should be allowed to fork it and add the changes if the original folks are taking a break or dont desire to mess with it anymore.
